backend/compact-connect/bin/create_staff_user.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+"""
+Staff user generation helper script. Run from `backend/compact-connect`.
+
+Note: This script requires the boto3 library and two environment variables:
+USER_POOL_ID=us-east-1_7zzexample
+USER_TABLE_NAME=Sandbox-PersistentStack-StaffUsersUsersTableB4F6C7C8-example
+
+The CLI must also be configured with AWS credentials that have appropriate access to Cognito and DynamoDB
+"""
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+USER_POOL_ID = os.environ['USER_POOL_ID']
+USER_TABLE_NAME = os.environ['USER_TABLE_NAME']
+
+
+cognito_client = boto3.client('cognito-idp')
+user_table = boto3.resource('dynamodb').Table(USER_TABLE_NAME)
+
+
+def create_compact_ed_user(*, username: str, email: str, compact: str):
+    print(f"Creating Compact ED user, '{username}', in {compact}")
+    sub = create_cognito_user(username=username, email=email)
+    user_table.put_item(
+        Item={
+            'pk': sub,
+            'createdCompactJurisdiction': f'{compact}/{compact}',
+            'permissions': {
+                compact: {
+                    'actions': {'read', 'admin'},
+                    'jurisdictions': {}
+                }
+            }
+        }
+    )
+
+
+def create_board_ed_user(*, username: str, email: str, compact: str, jurisdiction: str):
+    print(f"Creating Board ED user, '{username}', in {compact}/{jurisdiction}")
+    sub = create_cognito_user(username=username, email=email)
+    user_table.put_item(
+        Item={
+            'pk': sub,
+            'createdCompactJurisdiction': f'{compact}/{jurisdiction}',
+            'permissions': {
+                compact: {
+                    'actions': {'read'},
+                    'jurisdictions': {
+                        jurisdiction: {'actions': {'write', 'admin'}}
+                    }
+                }
+            }
+        }
+    )
+
+
+def create_cognito_user(*, username: str, email: str):
```

Comment:
The * forces the arguments to be keyword-only, so that our interface is more explicit in its use. Helps to prevent accidentally getting arguments out of order or confusion in the case of future revisions to the function signature.

Reply:
ah cool! the more you know.